bootstrap/secret/secret_test.go

```diff
@@ -81,7 +79,7 @@ func TestNewSecretProvider(t *testing.T) {
 					case "/v1/auth/token/lookup-self":
 						w.WriteHeader(http.StatusOK)
 						_, _ = w.Write([]byte(testTokenResponse))
-					case "/redisdb":
+					case "/v1/secret/edgex/testServiceKey//redisdb":
```

Comment:
remove one slash:
```suggestion
					case "/v1/secret/edgex/testServiceKey/redisdb":
```

Reply:
Nope, this is how it currently gets generated by code in go-mod-secrets. I tried to use single slash... :-(

Reply:
ok